tests/features/joinup_document/api.feature

```diff
@@ -6,13 +6,14 @@ Feature: Document API
 
   Scenario: Programmatically create a Custom Page
     Given the following collection:
-      | title             | Joinup document name |
-      | owner             | Joinup Derp          |
-      | logo              | logo.png             |
-      | moderation        | yes                  |
-      | closed            | yes                  |
-      | elibrary creation | facilitators         |
+      | uri               | http://joinup.eu/document_api_collection |
+      | title             | Joinup document name                     |
+      | owner             | Joinup Derp                              |
+      | logo              | logo.png                                 |
+      | moderation        | yes                                      |
+      | closed            | yes                                      |
+      | elibrary creation | facilitators                             |
     And document content:
-      | title    | field_document_short_title | body               | collection              |
-      | JD title | Short                      | Dummy description. | http://joinup.eu/jd/jde |
+      | title    | field_document_short_title | body               | og_group_ref                             |
+      | JD title | Short                      | Dummy description. | http://joinup.eu/document_api_collection |
```

Comment:
It is the same comment as above. This was a false positive. If you break the test, and see this in UI, it doesn't actually connect the entity to the collection.
And in the api.features we only test "And I have 1 item of this type" so it does not provide assurance about consistency of data.

Still, I will do something using a hook.